.github/workflows/trivy-image-scan.yml

```diff
@@ -78,6 +87,10 @@ jobs:
           scan-name: docker-${{ inputs.SCAN_NAME }}
           scan-ref: ${{ inputs.IMAGE_REF }}
           uses-java: ${{ inputs.USES_JAVA }}
+          allow-unfixed: ${{ inputs.SECURITY_ISSUES_BLOCK_ONLY_IF_FIX_AVAILABLE }}
+          remote-vex: |
+            telicent-oss/telicent-base-images
+            ${{ inputs.REMOTE_VEX }}
 
       - name: Generate SBOM for image (IF has tags for release)
         if: startsWith(github.ref, 'refs/tags/')
```

Comment:
- [ ] TODO Create a new ticket to fold SBOM generation functionality into the shared `telicent-oss/trivy-action` rather than having to call out to `aquasecurity/trivy-action` again

Reply:
CORE-776